frontend/components/card/discussion/CardDiscussionInput.vue

```diff
@@ -159,6 +159,7 @@ const editor = useEditor({
         class:
           "hover:underline font-bold rounded-2xl box-decoration-clone px-1 py-0.5",
       },
+      // @ts-expect-error Ignore mismatched types.
       suggestion: Suggestion,
```

Comment:
This was the one error that I wasn't sure on ignoring 🤔 If either of you have an idea on this, @annaleexjohnson or @GoodMorningA1i, then we'd welcome another PR to fix it!

Reply:
For now I'd like to bring this in :)